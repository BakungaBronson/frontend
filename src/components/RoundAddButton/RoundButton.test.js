import React from 'react';

import { shallow } from 'enzyme';

import RoundAddButton from './index';

describe('Button Component', () => {
  it('Renders a button component', () => {

    const buttonEvent = jest.fn();
    const ButtonComponent = shallow(<RoundAddButton className="RoundAddButton" onClick={buttonEvent} />);

    expect(ButtonComponent.find('.RoundAddButton').hasClass('RoundAddButton')).toBe(true);

    ButtonComponent.find('.RoundAddButton').simulate('click');
    expect(buttonEvent).toHaveBeenCalled();

    expect(ButtonComponent).toMatchSnapshot();

  });
});