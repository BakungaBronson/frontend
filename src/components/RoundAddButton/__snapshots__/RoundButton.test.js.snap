// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Renders a button component 1`] = `ShallowWrapper {}`;
